src/lesson10/Task2.java

```diff
@@ -0,0 +1,35 @@
+package lesson10;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Task2 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("enter the length of the array: ");
+        int lengthArray = scanner.nextInt();
+        int[] dataNumbers = new int[lengthArray];
```

Comment:
what data? and from the var type, I see it is the number, the name is not descriptive